integreat_cms/cms/views/imprint/imprint_form_view.py

```diff
@@ -92,25 +93,34 @@ def get(self, request, *args, **kwargs):
 
         disabled = False
         if imprint:
-            # Show information if latest changes are only saved as draft
+            # Show information if latest changes are only saved as draft, but there is an earlier public version of this translation
             public_translation = imprint.get_public_translation(language.slug)
-            if public_translation and imprint_translation != public_translation:
+            if (
+                public_translation
+                and imprint_translation != public_translation
+                and public_translation.id
+            ):
```

Comment:
```suggestion
            if public_translation and imprint_translation != public_translation:
```
Hmm, I don't really understand this change. `get_public_translation()` either returns `None`, or a translation object. If it returns a translation object, `id` will definitely be set, so I think in this context, the conditions `if public_translation` and `if public_translation.id` are equivalent... or am I missing something here?

Reply:
> If it returns a translation object, id will definitely be set

Not really.
When public_translation is not None AND public_translation.id is not None --> returned translation is in the requested language.
When public_translation is not None BUT public_translation.id is None --> returned translation is in fallback language.

Reply:
So with the new condition by public_translation.id we won't display the message, when public translation is a fallback translation, otherwise it produced incorrect link.

Reply:
Ahh ok got it, thanks for the explanation! And also thanks for the comment, I think this is very helpful so we don't accidentally remove this condition in the future.